app/src/main/java/swati4star/createpdf/activity/CropImageActivity.java

```diff
@@ -133,6 +133,7 @@ public void cropOKButtonClicked() {
         Bitmap crop = ivCrop.crop();
         mCropImageView.setImageBitmap(crop);
         setContentView(mMainView);
+        cropButtonClicked();
```

Comment:
`cropButtonClicked()` method inside `cropOKButtonClicked()` is little confusing!!! Can you explain please...

Reply:
Use the cropButtonClicked() to store the image with the URI. Because after you crop with a quadrilateral selection, the image should be saved immediately. If not saved, the URI of the image will be null and an exception will occur.

Reply:
cropButtonClicked() is actually a method to save the cropped image as a file.